bring_api/types.py

```diff
@@ -70,4 +70,16 @@ class BringNotificationType(Enum):
     GOING_SHOPPING = "GOING_SHOPPING"
     CHANGED_LIST = "CHANGED_LIST"
     SHOPPING_DONE = "SHOPPING_DONE"
-    URGENT_MESSAGE = "URGENT_MESSAGE"
\ No newline at end of file
+    URGENT_MESSAGE = "URGENT_MESSAGE"
+
+class BringItem(TypedDict):
```

Comment:
Duplicate to `BringPurchase`, could be refactored to `BringItem` as well.

Reply:
this endpoint expects `spec` as parameter but the endpoint to get the list items has `specifications` as node name in the json response. Don't know if this would be a problem.